### ISSUE TITLE
App randomly closes after a couple seconds.

### ISSUE THREAD

**Issue by Blackmaster117** [lubomyr/uae4arm#45]
Asus rog phone 8 pro
Android 14 (probably the problem)

**Comment by Stern77** [comment]
Same on my Samsung Galaxy S22 Ultra running on Android 14

**Comment by diejuse** [comment]
Same. Z Flip 5 and Pixel Fold. Android 14. Please, any fix?

**Comment by MASHtm** [comment]
I struggled with the same issue on my new S24, Android 14 ... I got it running stable again after recalibrating the touchscreen.

Open the SDL  config at start, go to "Mouse emulation" and "Calibrate touchscreen". I verified it by reseting the SDL config. After that it crashed again as before. Recalibrating fixed it again.
YMMV.

While calibrating do not stretch the field beyond the visible area (control key at right and camera area at left) to prevent mapping issues in the uae menu.

**Comment by diejuse** [comment]
> I struggled with the same issue on my new S24, Android 14 ... I got it running stable again after recalibrating the touchscreen.
> 
> Open the SDL config at start, go to "Mouse emulation" and "Calibrate touchscreen". I verified it by reseting the SDL config. After that it crashed again as before. Recalibrating fixed it again. YMMV.
> 
> While calibrating do not stretch the field beyond the visible area (control key at right and camera area at left) to prevent mapping issues in the uae menu.

It worked! Thank you very much!
But now I have another very important problem, which only occurs in new versions of Android. Maybe you can solve it.
Every time I touch the touch screen, a left mouse click occurs. This makes it impossible for me to use the app.

**Comment by MASHtm** [comment]
"slightly" OT. But it doesn't happen on my phone. Left MB and especially double click always was a PITA. usually I change the SDL config for it as well. if nothing else helps try "physical key". eg. volume up/down.

**Comment by diejuse** [comment]
> "slightly" OT. But it doesn't happen on my phone. Left MB and especially double click always was a PITA. usually I change the SDL config for it as well. if nothing else helps try "physical key". eg. volume up/down.

Unfortunately "physical key" doesn't work on modern Androids. I choose VOL+ or VOL- but it doesn't work.

**Comment by AndyC-cell** [comment]
> I struggled with the same issue on my new S24, Android 14 ... I got it running stable again after recalibrating the touchscreen.
> 
> Open the SDL config at start, go to "Mouse emulation" and "Calibrate touchscreen". I verified it by reseting the SDL config. After that it crashed again as before. Recalibrating fixed it again. YMMV.
> 
> While calibrating do not stretch the field beyond the visible area (control key at right and camera area at left) to prevent mapping issues in the uae menu.

Hi, thank you, this stopped the crashing. Is there any way to "save" this calibration cause when I reopen the app after closing it it starts crashing again but if I keep the app open after calibration it doesn't.

I'm on Xperia 5 V, Android 15 BTW

**Comment by MASHtm** [comment]
> Hi, thank you, this stopped the crashing. Is there any way to "save" this calibration cause when I reopen the app after closing it it starts crashing again but if I keep the app open after calibration it doesn't.

Yeah, I noticed that as well. Didn't find any fix/workaround for that yet.

**Comment by AndyC-cell** [comment]
> > Hi, thank you, this stopped the crashing. Is there any way to "save" this calibration cause when I reopen the app after closing it it starts crashing again but if I keep the app open after calibration it doesn't.
> 
> Yeah, I noticed that as well. Didn't find any fix/workaround for that yet.

Ah, ok, thank you for the quick reply

**Comment by Aydogano277** [comment]
> I struggled with the same issue on my new S24, Android 14 ... I got it running stable again after recalibrating the touchscreen.
> 
> Open the SDL config at start, go to "Mouse emulation" and "Calibrate touchscreen". I verified it by reseting the SDL config. After that it crashed again as before. Recalibrating fixed it again. YMMV.
> 
> While calibrating do not stretch the field beyond the visible area (control key at right and camera area at left) to prevent mapping issues in the uae menu.

I did the same way, but it closes again frequently. (Xiaomi Mi 14t pro with the last package update)

**Comment by Bastich-666** [comment]
I am having the same issue where UAE4ARM v1.0.1 randomly crashes. This occurs in game as well as in the UI. I am running it on an AYN Odin 3 which is using the Snapdragon Elite SOC. Is there any kind of debug I can send or help with?

**Comment by lubomyr** [comment]
App isn't compatible with latest android versions. It requires to rebuild
it with newest android sdk and android sdl.

нд, 25 січ. 2026 р., 14:23 користувач James ***@***.***> пише:

**Comment by joaocazevedo1-oss** [comment]
For me uae4arm is by far the best amiga emulator on android, you can do things like chose one folder for dh0 and others for dh1, dh2, dh3.....and they dont even need to be hdf files.
But even with this new version v1.0.4.4 it continues doing the same thing, the app closes after tapping on the screen.
My phone: Redmi Note 9S
Android: 12
MIUI: 14.0.4

**Comment by lubomyr** [comment]
> But even with this new version v1.0.4.4 it continues doing the same thing, the app closes after tapping on the screen.

I updated to 1.0.4.5, but not sure if it can help, as possible solution i can recommend to try:
- Reset app settings: via unistall app, then new install or you can try to reset this settings in startup SDL menu.
-  Also you can try to change advanced mouse emulation settings in startup sdl menu

**Comment by MASHtm** [comment]
Just as a sidenote... I do not have this issue since my S24 got Android 16

**Comment by joaocazevedo1-oss** [comment]
Hi lubomyr, many thanks for your latest fix [version 1.0.4.8] it works great, the app no ​​longer exits when you touch the screen.
My phone: Redmi Note 9S
Android: 12
MIUI: 14.0.4
